### PR TITLE
OpenVINO backend

### PR DESCRIPTION
**Pull request by dkurt**
Hello, Cellpose community!

This PR proposes Intel OpenVINO backend for deep learning inference.

Efficiency measured on https://drive.google.com/open?id=18syVlaix8cIlrnNF20pEWKMWUsKx9R9z (16 images):
```
python -m cellpose --dir cellpose_test --pretrained_model cyto --chan 2 --chan2 3
```
| Backend | Processing time |
| ---|---|
| PyTorch 1.10.0 | 85.903 sec |
| PyTorch 1.10.0 (mkldnn) | 94.450 sec |
| OpenVINO 2021.3 | 58.324 sec |

CPU: `Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz`

for tests check local PR: https://github.com/dkurt/cellpose/pull/1

### CODE REVIEW

**Comment by dkurt** [comment]
@kevinjohncutler, @carsen-stringer, may I ask you to review? Thanks in advane!

File: cellpose/core.py [hunk 1]
```diff
@@ -256,6 +262,9 @@ def eval(self, x, batch_size=8, channels=None, channels_last=False, invert=False
         nimg = len(x)
         self.batch_size = batch_size
 
+        if OPENVINO_ENABLED:
+            self.net = openvino_utils.to_openvino(self.net)
```
Comment (dkurt) [inline cellpose/core.py:266]: Should I add a flag to enable/disable OpenVINO?
```python
model = models.Cellpose(..., openvino=True)
```

**Comment by dkurt** [comment]
Hi there! Just wanted to ask if you had a chance to review this proposal.

**Comment by dkurt** [comment]
@kevinjohncutler, @carsen-stringer, sorry for pinging. May I ask for feedback if that proposal is actual for your project?

**Comment by dkurt** [comment]
Hi, @carsen-stringer! If it might take much maintenance effort, do you mind if I publish extensions as a separate package like `cellpose-openvino` which will depend on `cellpose`?

**Comment by dkurt** [comment]
Hi, @kevinjohncutler! I'm a bit confused that there is no response from dev team. It's interesting to see Cellpose through https://github.com/BIOP/qupath-extension-cellpose in [QuPath](https://github.com/qupath/qupath) tool to help digital pathology community democratize usage of heavy deep learning pipelines. I believe that direct contribution might be better than forking but if you have no objections, can we create an extension?

**Comment by kevinjohncutler** [comment]
@dkurt I'm terribly sorry for the delay on this. I'm trying to be better at checking my github notifications, and I've been going through the backlog this week. I agree that a direct contribution would be better than forking. In the meantime, please do feel free to create an extension. I'm really very interested in your pull request. This reduced runtime would be really useful. Again, super sorry that your pull request has been neglected (thank you for tagging us!), and I'll do my best to evaluate it soon.

**Comment by dkurt** [comment]
@kevinjohncutler, oh, sorry, no worries. I should email instead. I will also review PR once again to make it actual.

**Comment by carsen-stringer** [comment]
we are happy to add user contributions to the contrib folder, but not directly to the source code. if the changes you propose require that, please make a fork

**Comment by dkurt** [comment]
Thanks, @carsen-stringer! Opened a pull request to contrib folder: https://github.com/MouseLand/cellpose/pull/461

**Pull request closed without merging**